pkgs/top-level/all-packages.nix

```diff
@@ -4778,7 +4778,7 @@ with pkgs;
 
   recutils = callPackage ../tools/misc/recutils { };
 
-  recoll = callPackage ../applications/search/recoll { };
+  recoll = callPackage ../applications/search/recoll { withGui = true; };
```

Comment:
Why not just use `withGui ? true` in the arguments?

Reply:
no special reason, reading other examples in all-packages.nix gave me the impression it was the preferred  way. I can change is you prefer it

Reply:
Yeah I'd prefer that, otherwise the default value in the package definition doesn't even have any use and can only confuse.

Reply:
Updated :)